sources/core/Stride.Core.Mathematics/Vector2.cs

```diff
@@ -207,6 +207,24 @@ public float[] ToArray()
             return new float[] { X, Y };
         }
 
+        /// <summary>
+        /// Moves the first vector2 to the second one in a straight line.
+        /// </summary>
+        /// <param name="First">The first point.</param>
+        /// <param name="Second">The second point.</param>
+        /// <param name="MaxDistancePerCall">The rate at which the first point is going to move towards the second point.</param>
+        public static Vector2 Goto(Vector2 First, Vector2 Second, float MaxDistancePerCall)
```

Comment:
method input parameters should be lower case to match the convention in the stride source code.

Reply:
alright but should I make all of them lower case or just the M for MaxDistancePerCall?

Reply:
The first letter of all parameters should be lower case.